nix/eval-modules.nix

```diff
@@ -0,0 +1,16 @@
+rec {
+  evalModules = { pkgs, name, modules }: (pkgs.lib.evalModules {
+    specialArgs = {
+      inherit name pkgs;
+      process-compose-flake-lib = (import ./process-compose-flake-lib.nix) { lib = pkgs.lib; };
+    };
+    modules = [
+      ./process-compose
+    ] ++ modules;
+  });
+
+  makeProcessCompose = { pkgs, name, modules }: (evalModules {
```

Comment:
makeProcessCompose is basically the same as evalModules but returns the process directly for a better user experience, but if someone needs the full config, evalModules can be used.